### PR TITLE
Should restock

### PR DESCRIPTION
**Pull request by ericdcobb**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,83 +1,74 @@
...
-    productUrl.searchParams.append('organizationId', 'bfcadfb1-34df-40c1-acf6-1be6ba20de0f');
-    productUrl.searchParams.append('locationId', 'bf726b02-333a-448a-b800-e90f80a955be');
+    const defaultColDef = useMemo(() => ({
+        sortable: true
+    }), []);
 
-    const apiKey = localStorage.getItem('ARCHX_API_KEY');
-    const token = localStorage.getItem('ARCHX_TOKEN');
+    let organizationId = 'bfcadfb1-34df-40c1-acf6-1be6ba20de0f';
+    let locationId = 'bf726b02-333a-448a-b800-e90f80a955be';
```
Comment (mattpetters) [inline app/page.tsx:28]: total nitpick here but I generally always use const unless the values are going to change

Comment (ericdcobb) [reply app/page.tsx:38]: me too. I'm actually surprised the linter didn't bark about this, might need to further extend that.

**Comment by ericdcobb** [comment]
Screenshot 2023-10-31 at 4 47 27 PM

File: app/page.tsx [hunk 2]
```diff
@@ -1,92 +1,93 @@
...
-              <AgGridReact
-              rowData={rowData}
-              columnDefs={columnDefs as any}
-              defaultColDef={defaultColDef}
-              animateRows={true}
-              rowSelection='multiple'
-              onCellClicked={cellClickedListener}
-              />
+                height: 1000,
+                top: 0,
```
Comment (mattpetters) [inline app/page.tsx:70]: little lost diff here

File: app/page.tsx [hunk 3]
```diff
@@ -1,92 +1,93 @@
...
-  const cellClickedListener = useCallback( (event:any) => {
-    console.log('cellClicked', event);
-  }, []);
-
-  return (
-    <main className={`flex min-h-screen flex-col items-center bg-hca-white ${lexend.className}`}>
+    return (
+        <main className={`flex min-h-screen flex-col items-center bg-hca-white ${lexend.className}`}>
             <div className="flex flex-row w-full bg-hca-dark-blue divide-x-2 divide-solid">
-              <Link href="/" >
```
Comment (mattpetters) [inline app/page.tsx:65]: and here

**Comment by ericdcobb** [comment]
Screenshot 2023-10-31 at 4 57 21 PM

**Comment by ericdcobb** [comment]
Screenshot 2023-10-31 at 4 59 28 PM

**Review by mattpetters — APPROVED** [review]

**Pull request merged**